scrapy/commands/parse.py

```diff
@@ -7,7 +10,9 @@
 from scrapy.utils.conf import arglist_to_dict
 from scrapy.utils.spider import iterate_spider_output, spidercls_for_request
 from scrapy.exceptions import UsageError
-from scrapy import log
+
+logger = logging.getLogger('scrapy')
```

Comment:
Here's an example of the output using `__name__` loggers: https://gist.github.com/Curita/00ddfbddc28cd6f11934. I don't think there are many longer paths than `scrapy.contrib.downloadermiddleware.redirect` on Scrapy.